### ISSUE TITLE
error IDE0055 - seems inconsistent running in Devops

### ISSUE THREAD

**Issue by DerekSMorin** [dotnet/roslyn#61042]
I tried "Enforce code style on build ( experimental )".

I managed to clean up my errors, create a PR and put it through.

Then a subsequent CI build ended up failing with some formatting errors.

I put a new PR through ( kind of guessing at what was wrong in the file ) and managed to get that PR through and a regular CI pipelin build passing.

Now I'm seeing more code formatting errors on a test run that runs nightly.  Compiling as part of "dotnet test".

Here is the output of the failing test run:

`Active code page: 65001
Info: .NET Core SDK/runtime 2.2 and 3.0 are now End of Life(EOL) and have been removed from all hosted agents. If you're using these SDK/runtimes on hosted agents, kindly upgrade to newer versions which are not EOL, or else use UseDotNet task to install the required version.
e:\vsts\a\_tool\dotnet\dotnet.exe test e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\Bentley.Interoperability.APMTests.csproj --logger trx --results-directory e:\vsts\a\_temp
  Determining projects to restore...
  Restored e:\vsts\a\8989\s\src\Bentley.Interoperability.APM\Bentley.Interoperability.APM.csproj (in 7.71 sec).
  Restored e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\Bentley.Interoperability.APMTests.csproj (in 7.71 sec).
  Bentley.Interoperability.APM -> e:\vsts\a\8989\s\src\Bentley.Interoperability.APM\bin\Debug\Bentley.Interoperability.APM.dll
e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\AssetChangeRequestTests\AssetPrioritization\ProcessAssetProducerComponentTests.cs(68,115): error IDE0055: Fix formatting [e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\Bentley.Interoperability.APMTests.csproj]
e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\AssetChangeRequestTests\AssetPrioritization\ProcessAssetProducerComponentTests.cs(69,96): error IDE0055: Fix formatting [e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\Bentley.Interoperability.APMTests.csproj]
e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\RunTestsInParallel.cs(1,69): error IDE0055: Fix formatting [e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\Bentley.Interoperability.APMTests.csproj]
e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\RunTestsInParallel.cs(3,82): error IDE0055: Fix formatting [e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\Bentley.Interoperability.APMTests.csproj]
##[error]Error: The process 'e:\vsts\a\_tool\dotnet\dotnet.exe' failed with exit code 1
##[warning]No test result files were found.
##[warning].NET 5 has some compatibility issues with older Nuget versions(<=5.7), so if you are using an older Nuget version(and not dotnet cli) to restore, then the dotnet cli commands (e.g. dotnet build) which rely on such restored packages might fail. To mitigate such error, you can either: (1) - Use dotnet cli to restore, (2) - Use Nuget version 5.8 to restore, (3) - Use global.json using an older sdk version(<=3) to build
Info: Azure Pipelines hosted agents have been updated and now contain .Net 5.x SDK/Runtime along with the older .Net Core version which are currently lts. Unless you have locked down a SDK version for your project(s), 5.x SDK might be picked up which might have breaking behavior as compared to previous versions. You can learn more about the breaking changes here: https://docs.microsoft.com/en-us/dotnet/core/tools/ and https://docs.microsoft.com/en-us/dotnet/core/compatibility/ . To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting
##[error]Dotnet command failed with non-zero exit code on the following projects : e:\vsts\a\8989\s\tests\Bentley.Interoperability.APMTests\Bentley.Interoperability.APMTests.csproj
Finishing: Run Tests
`
Here is the smaller file it is complaining about:

// Copyright (c) Bentley Systems, Incorporated. All rights reserved.

// Only run tests in parallel in Release - see AssemblySetup.cs if modifying this
#if !DEBUG
[assembly: Parallelizable(ParallelScope.Fixtures)]
#endif

**Comment by 0liver** [comment]
Could it be that you have `LF` line endings in the lines mentioned in the error list? We're having an issue with `IDE0055` triggering on a file with `LF` line endings, which does not trigger when using `CRLF`.

**Comment by DAud-IcI** [comment]
If it's related to `LF` line endings as @0liver suggested, then it's a duplicate of #55526.

**Comment by DerekSMorin** [comment]
@0liver and @DAud-IcI 

As far as I can tell at least locally I have CRLF.

But looking at the file the line / characters in the error messages seem to be at the end of line.

It could be possible that the file has different line endings in the DevOps repo?

![image](https://user-images.githubusercontent.com/7568856/168401614-32d337f4-5679-4147-8a57-b5a5afef617b.png)

This image shows that the end of line 1 is 1,69.
The end of line 3 is 3,82

You can see vscode showing it as CRLF but I think things can vary with git.

**Comment by DerekSMorin** [comment]
I also don't know if the bug could be having a blank line between my two commented lines but line endings in git seems more likely.

**Comment by DerekSMorin** [comment]
@0liver , @DAud-IcI - I downloaded the file from DevOps and can see the line endings are LF.  Not sure why I only get a few errors instead of tons of errors for many files.  Shrug.

![image](https://user-images.githubusercontent.com/7568856/168401918-4755e9bb-80cf-4dcb-ae14-03c403a4010d.png)

**Comment by vatsalyaagrawal** [comment]
Duplicate of #55526
